java-frontend/src/main/java/org/sonar/java/UCFGJavaVisitor.java

```diff
@@ -362,6 +362,10 @@ private static boolean isObject(Type type) {
     return type.isSubtypeOf("java.lang.Object");
   }
 
+  private static Expression fullyQualifiedNameExpression(Type type) {
```

Comment:
Given the usage, I would simply inline this one, the method does not bring any added value.